php_memcached.c

```diff
@@ -3895,9 +3895,14 @@ PHP_METHOD(MemcachedServer, on)
 
 #endif
 
+#if PHP_VERSION_ID < 80200
```

Comment:
this should be defined only for 8.0/8.1, but needed for 7.x because of https://github.com/php/php-src/issues/8842